conductr_cli/resolvers/resolvers_util.py

```diff
@@ -9,7 +8,6 @@ def is_local_file(uri, require_bundle_conf):
     return parsed.scheme == 'file' and os.path.exists(parsed.path) and (
         os.path.isfile(parsed.path) or (
             os.path.isdir(parsed.path) and
-            detect_format_dir(parsed.path) is not None and
```

Comment:
This line was there so that `conduct load` would only call `bndl` on its argument if it was a directory that could reliably be turned into a bundle. A manual test for this would be to start the sandbox when inside the `conductr` project (that is, the git repo for ConductR itself that has folders such as conductr-elasticsearch, conductr-haproxy, etc).

I suspect this PR may result in this test failing because it autodetects any directory as a bundle, and thus `conduct load` won't resolve `conductr-haproxy` etc and will instead call `bndl` on it.

Related to https://github.com/typesafehub/conductr-cli/pull/469/files#diff-2b9533f9a56bd49b8daa989bdcf9e5d3R39